genestack_client/cla.py

```diff
@@ -191,12 +191,24 @@ def create_merged_reads(self, sources_folder, grouping_key):
 
 # preprocess microarrays
 
-class AffymetrixMicroarraysNormalisationApplication(CLApplication):
```

Comment:
We cannot remove existing classes. We should deprecate them first. I will try to find example.